### DISCUSSION TITLE
`_tkinter.TclError: image "pyimage4" doesn't exist`

### DISCUSSION THREAD

**Discussion by kennart** [TomSchimansky/CustomTkinter#1622] [General]
# Sir please actually the code is a huge GUI i'm developing before this error `_tkinter.TclError: image "pyimage4" doesn't exist` started popping up. And this error occur when i logout from  admin dashboard back to the homepage.

# I upload the portion of the code that is generating the error on to my github   https://github.com/kennart/customtkinter.git
# so that it can be easy for you to clone and look at the error for me.🙏🙏

# Please you can open it in sublime text3   (because of image path in visual studio code)
![code](https://github.com/TomSchimansky/CustomTkinter/assets/130964252/c0bbdce4-1022-4a7b-9d48-e05627449610)

**Comment by ghost** [comment]
Mr Tom Please i'm still looking up to you about this problem `_tkinter.TclError: image "pyimage4" doesn't exist`
because after doing several research using google and CHATGPT i came across this 👇👇

 It seems that the CTkImage class in the customtkinter library doesn't have an image argument in its constructor.
Given this limitation, it might not be possible to create a direct reference to the get_ctk_image function in a way that prevents the returned CTkImage object from being garbage collected. The handling of the CTkImage objects might be specific to the customtkinter library, and the library itself may manage the lifetime and garbage collection of these objects.
